tool_commands/guess_indicators/test_guess_indicators.py

```diff
@@ -0,0 +1,238 @@
+    if pump_bool or outlet_bool:
+        manholes_id_indicator = get_pump_or_outlet_manholes(session, ids)
+    if all_bool:
+        manholes_id_indicator = get_all_manholes(session)
+    return manholes_id_indicator
+
+
+def get_all_pipes(session):
+    pipes = {}
+    # only return manhole
+    sql_pipes = session.query(Pipe)
+    for pipe in sql_pipes:
+        # fill dict
+        pipes[pipe.id] = (pipe.friction_type, pipe.friction_value, pipe.material)
+    return pipes
+
+
+# def get_all_pipes_friction_type(session):
+#     pipes_id_friction_type = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_type[pipe.id] = pipe.friction_type
+#     return pipes_id_friction_type
+#
+#
+# def get_all_pipes_friction_value(session):
+#     pipes_id_friction_value = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_value[pipe.id] = pipe.friction_value
+#     return pipes_id_friction_value
+
+
+def empty_manhole_indicator(session, manholes_pre_empty):
+    """ empty column manhole_indicator (set is to NULL) of table v2_manhole """
+    manholes_ids = list(manholes_pre_empty)
+    up = (
+        update(Manhole)
+        .where(Manhole.id.in_(manholes_ids))
+        .values(manhole_indicator=None)
+    )
+    session.execute(up)
+    session.commit()
+
+
+def empty_pipe_friction_value(session, pipes_pre_empty):
+    """ empty column manhole_indicator (set is to NULL) of table v2_manhole """
+    pipes_ids = list(pipes_pre_empty)
+    up = (
+        update(Pipe)
+        .where(Pipe.id.in_(pipes_ids))
+        .values(friction_value=None, friction_type=None)
+    )
+    session.execute(up)
+    session.commit()
+
+
+@pytest.fixture()
+def tmp_sqlite_path(tmpdir):
+    sqlite_filename = "v2_bergermeer.sqlite"
+    orig_sqlite_path = TEST_DATA_DIR.joinpath(
+        "testmodel", "v2_bergermeer", sqlite_filename
+    )
+    tmp_sqlite_dir = Path(tmpdir)
+    tmp_sqlite_path = tmp_sqlite_dir.joinpath(sqlite_filename)
+    shutil.copy2(orig_sqlite_path, tmp_sqlite_path)
+    return tmp_sqlite_path
+
+
+@pytest.fixture()
+def db(tmp_sqlite_path):
+    db_type = "spatialite"
+    db_set = {"db_path": tmp_sqlite_path}
+    db = ThreediDatabase(db_set, db_type)
+    return db
+
+
+@pytest.fixture()
+def guesser(db):
+    return Guesser(db)
+
+
+def test_guess_manhole_indicator(db, guesser):
+    session = db.get_session()
+
+    # before we empty manholes, first get their [(id, manhole_indicator)]
+    manholes_pumps_pre_empty = get_manholes(session, pump_bool=True)
+    manholes_outlets_pre_empty = get_manholes(session, outlet_bool=True)
+    all_manholes_pre_empty = get_manholes(session, all_bool=True)
+    if not all_manholes_pre_empty:
+        # skip this test as there are no manholes in sqlite..
```

Comment:
Het is toch onze eigen test-sqlite? Dan weten we toch of er wel/niet manholes in zitten?